commit 56d066c36fe6511a179679a48259c5f1094283a7 (refs/changes/25/1925/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-05-10T09:44:28-07:00 (2 years, 5 months ago)
    
    TRITON-4 stop doing 'redis' VM image builds
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/targets.json.in b/targets.json.in
index 008f05c..66c6015 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -53,7 +53,6 @@ function build_headnode_target()
       "papi",
       "${platform}",
       "rabbitmq",
-      "redis",
       "sapi",
       "sdc",
       "sdcadm",
@@ -288,29 +287,6 @@ cat <<EOF
       {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
     ]
   },
-  "redis": {
-    "appliance": "true",
-    "build_platform": "20141030T081701Z",
-    "image_name": "redis",
-    "image_description": "SDC Redis",
-    "image_version": "1.0.0",
-    "image_uuid": "fd2cc906-8938-11e3-beab-4359c665ac99",
-    "pkgsrc": [],
-    "repos": [
-      {"url": "git@github.com:joyent/sdc-redis.git"}
-    ],
-    "public": true,
-    "deps": [
-      "amon",
-      "config-agent",
-      "registrar"
-    ],
-    "tarballs": [
-      {"tarball": "registrar/registrar-pkg-*.tar.bz2", "sysroot": "/"},
-      {"tarball": "amon/amon-agent-*.tgz", "sysroot": "/opt"},
-      {"tarball": "config-agent/config-agent-*.tar.bz2", "sysroot": "/opt/smartdc"}
-    ]
-  },
   "amonredis": {
     "appliance": "true",
     "build_platform": "20141030T081701Z",
